packages/react/src/components/pagination/pagination.tsx

```diff
@@ -173,22 +188,26 @@ export const Pagination: VoidFunctionComponent<PaginationProps> = ({
         );
     });
 
+    let pageStartIndex;
+    let pageEndIndex;
+    const pageSize = numberOfResults !== undefined ? Math.ceil(numberOfResults / totalPages) : undefined;
+    if (pageSize !== undefined && numberOfResults !== undefined) {
+        pageStartIndex = Math.max(1, pageSize * (currentPage - 1));
+        pageEndIndex = Math.min(numberOfResults, Math.floor(pageSize * (currentPage)));
+    }
+
     return (
         <Container className={className} isMobile={isMobile}>
-            {numberOfResults !== undefined && (
-                <ResultsLabel isMobile={isMobile} data-testid="resultsLabel">
-                    {`${numberOfResults} ${t('results')}`}
-                </ResultsLabel>
-            )}
-            <Navigation aria-label="pagination">
-                {firstLastNavActive && (
-                    <NavButton
-                        data-testid="firstPageButton"
-                        iconName="chevronsLeft"
-                        label="first page"
-                        enabled={canNavigatePrevious}
-                        onClick={() => changePage(1)}
-                    />
+            <Navigation aria-labelledby="pagination">
```

Comment:
Ca ne devrait pas etre hardcode, sinon ca implique que le user doit avoir un `<span id="pagination">Search results</span>` dans leur app. Aussi ca ne marcherais pas si t'a 2 pagination dans la meme page.

Faudrait que le user puisse le definir via les Props.

Reply:
La demande était de remplacer le aria-label par un aria-labelledby. C'était déjà hardcodé avec le aria-label, mais je suis d'accord avec toi. Il faudrait que ça soit customizable via prop. 

```
<Navigation aria-label="pagination">
// 
<Navigation aria-labelledby="pagination">

```

Reply:
Ouin mais l'attribut `aria-label` contient le label lui-même, alors que `aria-labelledby` contient un id qui référence un autre élément

Reply:
ça veut tu dire qu'on va vouloir rendre ça obligatoire 🤔 

aria-label can be used in cases where text that could label the element is not visible. If there is visible text that labels an element, use aria-labelledby instead.

Reply:
Comme c'est interne au component, ça ne serait pas vraiment nécessaire d'exposer ça dans une prop. On pourrait simplement s'en générer un avec `useId()`.

Reply:
Sauf que aria-labelledby doit avoir le ID d'un autre element html qui existe dans ta page, c'est pas un truc qui se genere. 

A part si l'element qu'on link est dans le component meme, sinon faut que le user nous dise c'est quoi son ID.

Reply:
Yep, il est dans le component même, c'est le texte de la pagination (si j'ai bien interprété la doc) 😄

Reply:
Tiens je crois t'a raison... dans ce cas on pourrait utiliser le useId() et ajouter le id sur CurrentPageLabelHeading

Reply:
Okey si je comprends bien la navigation est "aria-labelledBy" le heading "Pagination - 1-10 of 30 results" qui est déjà dans le component. Je dois ajouter le id dans le heading et on veut aussi générer un id spécial pour éviter des collisions.